js/components/form/base-form.js

```diff
@@ -136,7 +136,7 @@ export default {
                 }
             },
             errorPlacement: function(error, element) {
-                $(element).closest('.form-group,.field-wrapper').append(error);
+                $(element).closest('.form-group,.field-wrapper').children('div').append(error);
```

Comment:
Are you sure ? I think the extra `<div/>` breaks the bootstrap style: http://getbootstrap.com/css/#forms-control-validation